src/ejabberd_hooks.erl

```diff
@@ -128,7 +129,7 @@ run(Hook, Args) ->
           Host :: jid:server() | global,
           Args :: [any()]) -> ok.
 run(Hook, Host, Args) ->
```

Comment:
Maybe we should drop this function as it's deprecated and won't fit mongoose_acc's future form?

Reply:
Yes! It's confusing to see this in a codebase, since hook handlers receive accumulators anyway.